File: tests/cases/lock/locked/lua/reformat/inputs/src/billing/invoice.lua
-- An unrelated declaration inserted ABOVE the locked function.
local UNRELATED = 1

local Invoice = {}
Invoice.__index = Invoice

function Invoice.new(id)
  local self = setmetatable({}, Invoice)
  self.id = id
  return self
end

---@cdec locked(reason = "invoice wording is contractual")
function Invoice:formatted()
  -- a new explanatory comment
  return self.id
end

return Invoice

File: tests/cases/lock/locked/julia/reformat/inputs/src/billing/Invoice.jl
module Billing

using CdecRules

# An unrelated declaration inserted ABOVE the locked function.
const UNRELATED = 1

struct Invoice
    id::String
end

@locked reason="invoice wording is contractual" function formatted(inv::Invoice)::String
    # a new explanatory comment
    return inv.id
end

end
